push-module/push.js

```diff
@@ -0,0 +1,24 @@
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+const {
+  getCommitData,
+  getBuildData
+} = require('../lib');
+
+async function buildMetadata(input) {
+
+  const { stdout, stderr } = await exec(`npm v "${input.id}@${input.version}" --json`);
+  const data = JSON.parse(stdout);
```

Comment:
this should be used as the "module" field in the output object rather than the root object